app.py

```diff
@@ -1,3 +1,7 @@
+'''
+  A project made by asutosh97 and shibasisp as a part of pybites challenges. #lotsoffun #cloudyday
```

Comment:
Remove #cloudydays

Reply:
that day, it was very much cloudy, I even sent you pics from my balcony.. Its just a memory... let it be there..

Reply:
No.. change it.